### ISSUE TITLE
Having problems using the plugin

### ISSUE THREAD

**Issue by matansh** [Azure/fluentd-plugin-mdsd#51]
Hey, I am running in a ubuntu 16.04 docker and get this error

```
root@dc35eb97f143:/# /opt/td-agent/embedded/bin/fluent-gem install fluent-plugin-mdsd-0.1.5.master.49-td.amd64.gem
Successfully installed fluent-plugin-mdsd-0.1.5.pre.build.master.49
Parsing documentation for fluent-plugin-mdsd-0.1.5.pre.build.master.49
Installing ri documentation for fluent-plugin-mdsd-0.1.5.pre.build.master.49
Done installing documentation for fluent-plugin-mdsd after 0 seconds
1 gem installed
root@dc35eb97f143:/# /usr/sbin/td-agent --dry-run
2018-01-31 15:04:58 +0000 [info]: parsing config file is succeeded path="/etc/td-agent/td-agent.conf"
2018-01-31 15:04:58 +0000 [info]: starting fluentd-1.0.2 as dry run mode ruby="2.4.2"
/opt/td-agent/embedded/lib/ruby/gems/2.4.0/gems/fluent-plugin-mdsd-0.1.6.pre.build.master.54/lib/fluent/plugin/out_mdsd.rb:14:in `require_relative': /opt/td-agent/embedded/lib/ruby/gems/2.4.0/gems/fluent-plugin-mdsd-0.1.6.pre.build.master.54/lib/fluent/plugin/Liboutmdsdrb.so: undefined symbol: rb_cFixnum -
 /opt/td-agent/embedded/lib/ruby/gems/2.4.0/gems/fluent-plugin-mdsd-0.1.6.pre.build.master.54/lib/fluent/plugin/Liboutmdsdrb.so (LoadError)
        from /opt/td-agent/embedded/lib/ruby/gems/2.4.0/gems/fluent-plugin-mdsd-0.1.6.pre.build.master.54/lib/fluent/plugin/out_mdsd.rb:14:in `initialize'
        from /opt/td-agent/embedded/lib/ruby/gems/2.4.0/gems/fluentd-1.0.2/lib/fluent/plugin.rb:149:in `new'
        from /opt/td-agent/embedded/lib/ruby/gems/2.4.0/gems/fluentd-1.0.2/lib/fluent/plugin.rb:149:in `new_impl'
        from /opt/td-agent/embedded/lib/ruby/gems/2.4.0/gems/fluentd-1.0.2/lib/fluent/plugin.rb:104:in `new_output'
        from /opt/td-agent/embedded/lib/ruby/gems/2.4.0/gems/fluentd-1.0.2/lib/fluent/agent.rb:128:in `add_match'
        from /opt/td-agent/embedded/lib/ruby/gems/2.4.0/gems/fluentd-1.0.2/lib/fluent/agent.rb:72:in `block in configure'
        from /opt/td-agent/embedded/lib/ruby/gems/2.4.0/gems/fluentd-1.0.2/lib/fluent/agent.rb:64:in `each'
        from /opt/td-agent/embedded/lib/ruby/gems/2.4.0/gems/fluentd-1.0.2/lib/fluent/agent.rb:64:in `configure'
        from /opt/td-agent/embedded/lib/ruby/gems/2.4.0/gems/fluentd-1.0.2/lib/fluent/root_agent.rb:112:in `configure'
        from /opt/td-agent/embedded/lib/ruby/gems/2.4.0/gems/fluentd-1.0.2/lib/fluent/engine.rb:131:in `configure'
        from /opt/td-agent/embedded/lib/ruby/gems/2.4.0/gems/fluentd-1.0.2/lib/fluent/engine.rb:96:in `run_configure'
        from /opt/td-agent/embedded/lib/ruby/gems/2.4.0/gems/fluentd-1.0.2/lib/fluent/supervisor.rb:770:in `run_configure'
        from /opt/td-agent/embedded/lib/ruby/gems/2.4.0/gems/fluentd-1.0.2/lib/fluent/supervisor.rb:553:in `dry_run'
        from /opt/td-agent/embedded/lib/ruby/gems/2.4.0/gems/fluentd-1.0.2/lib/fluent/supervisor.rb:540:in `dry_run_cmd'
        from /opt/td-agent/embedded/lib/ruby/gems/2.4.0/gems/fluentd-1.0.2/lib/fluent/supervisor.rb:475:in `run_supervisor'
        from /opt/td-agent/embedded/lib/ruby/gems/2.4.0/gems/fluentd-1.0.2/lib/fluent/command/fluentd.rb:310:in `<top (required)>'
        from /opt/td-agent/embedded/lib/ruby/site_ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
        from /opt/td-agent/embedded/lib/ruby/site_ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
        from /opt/td-agent/embedded/lib/ruby/gems/2.4.0/gems/fluentd-1.0.2/bin/fluentd:8:in `<top (required)>'
        from /opt/td-agent/embedded/bin/fluentd:22:in `load'
        from /opt/td-agent/embedded/bin/fluentd:22:in `<top (required)>'
        from /usr/sbin/td-agent:7:in `load'
        from /usr/sbin/td-agent:7:in `<main>'
```

Thanks in advance for any help :)

**Comment by sbonebrake** [comment]
I see this as well when trying to use this gem with fluentd 1.0. But also because there isn't a published release of this gem, I unpacked the gem, changed the Gemfile to use fluentd ~> 1.0 and manually did the most recent logger fix.
Are you doing anything to make this 1.0 compatible?
@julioct Did you hit this when trying to use fluentd 1.0?

**Comment by julioct** [comment]
There is no gem (that I know of) that includes the necessary fix for FluentD 1.0 compatibility. You would need to build the plugin yourself at commit 123a8e292531fa66a664de0babdf1d49823cd7ae to get a compatible build.

@jasonzio Can we get a new release/gem?
